.github/workflows/ci.yml

```diff
@@ -440,18 +393,17 @@ jobs:
       matrix: ${{ fromJson(needs.build-test-matrix.outputs.matrix) }}
     timeout-minutes: 60
     steps:
-      - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0 # checkout all commits to be able to determine merge base for GIB
-      - name: Fetch base ref to find merge-base for GIB
-        run: .github/bin/git-fetch-base-ref.sh
-      - uses: actions/setup-java@v3
+      - uses: trinodb/github-actions/checkout-and-setup@6a40cfbfa0ad484d69f5482b29c41fbaac72944e
+      - name: Link workflow
```

Comment:
This should not be necessary as it is performed by the `update-check` action. Looks like a leftover.